app/workers/document_list_export_worker.rb

```diff
@@ -46,4 +49,9 @@ def generate_csv(format, query)
       end
     end
   end
+
+  def upload_csv(filename, csv)
+    s3_file = S3FileUploader.save_file_to_s3(filename, csv)
```

Comment:
This doesn't exist right? At least not at this commit.

Reply:
Oh yeah. I added it in the next commit but actually those should be squashed together.